classes/extSuperDirtStartup.sc

```diff
@@ -14,7 +14,10 @@
 		// boot the server and start SuperDirt
 		server.waitForBoot {
 			~dirt = SuperDirt(numChannels, server);
-			~dirt.loadSoundFiles(path);   // load samples (path can be passed in)
```

Comment:
Yes. What I found was that when sending a path into SuperDirt.start, it would only load the samples on the path I supplied and did not load the default samples as well.  My change here ensures that the default samples are loaded whether or not a user supplies another path.